subsystems/drivesubsystem.py

```diff
@@ -479,7 +479,7 @@ def applyStates(self, moduleStates: Tuple[SwerveModuleState]) -> None:
         self.backRightModule.applyState(backRightState)
 
     def getRotation(self) -> Rotation2d:
-        return Rotation2d.fromDegrees(self.gyro.getYaw())
+        return Rotation2d.fromDegrees(-self.gyro.getYaw())
```

Comment:
Is this why we had issues on Sunday?

Reply:
Yep, Wpilib went from CW+ to CCW+ in a few key areas